notebook2/table.py

```diff
@@ -0,0 +1,71 @@
+import base64
+import mysql.connector
+import kubernetes as kube
+import os
+
+if 'BATCH_USE_KUBE_CONFIG' in os.environ:
+    kube.config.load_kube_config()
+else:
+    kube.config.load_incluster_config()
+k8s = kube.client.CoreV1Api()
+
+SQL_HOST_DEF = os.environ.get('SQL_HOST')
+
+
+class Table:
+    @staticmethod
+    def getSecret(b64str):
+        return base64.b64decode(b64str).decode('utf-8')
+
+    @staticmethod
+    def getSecrets():
+        secrets = {}
+
+        try:
+            res = k8s.read_namespaced_secret('get-users', 'default')
+            data = res.data
+
+            if SQL_HOST_DEF is not None:
+                host = SQL_HOST_DEF
+            else:
+                host = Table.getSecret(data['host'])
+
+            secrets['user'] = Table.getSecret(data['user'])
+            secrets['password'] = Table.getSecret(data['password'])
+            secrets['db'] = Table.getSecret(data['db'])
+            secrets['host'] = host
+        except kube.client.rest.ApiException as e:
+            print(e)
+
+        return secrets
+
+    def __init__(self):
+        secrets = Table.getSecrets()
+
+        if not secrets:
+            raise "Couldn't read get-users secret"
+
+        self.cnx = mysql.connector.connect(**secrets)
+
+    def get(self, user_id):
+        cursor = self.cnx.cursor(dictionary=True)
+
+        cursor.execute(
```

Comment:
I also wrote a Table class for Batch #5572. Not sure if it's worth trying to unify them at some point, but just wanted to make sure you were aware.

Reply:
Yeah, I was, thanks. I think it may be worth inheriting from it at some point (this class is quite simple at the moment). If you all wanted this sooner, I could do that, probably in conjunction with a move to aiohttp, since your Database class relies on asyncio.